### ISSUE TITLE
Scheduled Transactions

### ISSUE THREAD

**Issue by Ga68** [andreroggeri/ynab-sdk-python#137]
I really appreciate all the work that went into this. Thanks so much!

I think that Scheduled Transactions (future dates) aren't yet available based on what I read and what I can try to see digging into the code.

¿Is that right?

Looks like they're different from Transactions, though clearly similar.

**Comment by andreroggeri** [comment]
Yeah, according with the [API spec,](https://api.ynab.com/v1#/Scheduled%20Transactions) there's no endpoint for creating scheduled/future transactions :(


I asked for support, and they directed me to the [feature request form](https://youneedabudget.typeform.com/to/Pt6cek?typeform-source=support.ynab.com). If we can get more people asking for this support from the API, we might get them to do it.

**Comment by Ga68** [comment]
understood - thanks. there seems to be one for reading at least. ¿so would that, in theory, be possible to do?

**Comment by Ga68** [comment]
I'm working on a pr to at least get scheduled transactions
https://github.com/andreroggeri/ynab-sdk-python/compare/main...Ga68:ynab-sdk-python:scheduled_transactions?expand=1

I still need to work out the test cases, but the retrieval of the data works.

**Comment by mosius-chwy** [comment]
@andreroggeri 
support has been added

https://api.ynab.com/v1#/Scheduled%20Transactions/createScheduledTransaction

**Comment by Ga68** [comment]
@andreroggeri 
support has been added
https://api.ynab.com/v1#/Scheduled%20Transactions/createScheduledTransaction
